### ISSUE TITLE
Cannot turn off "checkSyntaxOnSave" option

### ISSUE THREAD

**Issue by amorphius** [xinchaobeta/compile-selected-es6#2]
Here is my User config

```
{
  "envPATH": "/usr/local/bin",
  "stage": 0,
  "checkSyntaxOnSave": false
}
```

But it keep checking syntax on save

**Comment by xinchaobeta** [comment]
I have fixed it in v0.2.5 https://github.com/xinchaobeta/compile-selected-es6/commit/57e0ce746054b124b39f72763510f0796da96870 (The website packagecontrol.io may take some time to sync, so you should wait it to be updated)

Your config is ok

**Comment by xinchaobeta** [comment]
https://github.com/xinchaobeta/compile-selected-es6/commit/57e0ce746054b124b39f72763510f0796da96870
